### ISSUE TITLE
Obscuring character and other values missing

### ISSUE THREAD

**Issue by Gafielt** [lzhuor/auto_size_text_field#10]
It would be nice if you could implement the obscure character field, as well as all other missing inputs to the Textfield.

**Comment by lzhuor** [comment]
@Gafielt Could you please share some more examples of the missing params of `TextField` ? Thanks

**Comment by Gafielt** [comment]
yeah sure. I'm not sure what all these are for, but I still think it is useful for the sake of completeness.

- autofillHints
- cursorheight
- mouseCursor
- obscuringCharacter
- onAppPrivateCommand
- restorationId

Flutter Version 10.0.18363.1198
